presto-docs/src/main/sphinx/connector/jmx.rst

```diff
@@ -45,11 +62,27 @@ The following query shows the open and maximum file descriptor counts
 for each node::
 
     SELECT openfiledescriptorcount, maxfiledescriptorcount
-    FROM jmx.jmx."java.lang:type=operatingsystem";
+    FROM jmx.current."java.lang:type=operatingsystem";
 
 .. code-block:: none
 
      openfiledescriptorcount | maxfiledescriptorcount
     -------------------------+------------------------
                          329 |                  10240
     (1 row)
+
+The second schema is called ``history`` and contains dumped tables as configured
+in ``jmx.properties``. It contains ``dump-tables`` with exactly the same
```

Comment:
Technically, the properties file could be named anything. I'd say something like:

The ``history`` schema contains the list of tables configured in the connector properties file.
The tables have the same columns as those in the ``current`` schema, but with an additional 
timestamp column ...